mobile/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,9 @@
     <string name="settings_openhab_icon_format_value_svg" translatable="false">SVG</string>
     <string name="settings_openhab_icon_format_png">Bitmap</string>
     <string name="settings_openhab_icon_format_svg">Vector</string>
+    <string name="settings_image_widget_scaling">Scale images to fit screen</string>
+    <string name="settings_image_widget_scaling_summary_off">Only scale images down \'Image\' widgets when they\'re larger than the screen, but don\'t scale them up to match the screen size</string>
```

Comment:
```suggestion
    <string name="settings_image_widget_scaling_summary_off">Don\'t scale \'Image\' widgets up to match the screen size</string>
```
I don't think any user would expect to see the image only partly, because it's too large.

Reply:
Reworded it slightly differently, but keeping the gist of the suggestion.